package.json

```
@@ -90,6 +98,8 @@
       "packages/fluentui/*"
     ],
     "nohoist": [
+      "packages/web-components/@storybook",
+      "packages/web-components/@storybook/**",
```

I think you only need this one line since `@storybook` isn't a package itself
```suggestion
      "packages/web-components/@storybook/**",
```

---

good catch. ty